app/build.gradle

```diff
@@ -28,6 +39,9 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'androidx.recyclerview:recyclerview:1.1.0-beta04'
     testImplementation 'junit:junit:4.12'
+    testImplementation "androidx.room:room-testing:$roomVersion"
+    kapt "androidx.room:room-compiler:$roomVersion"
+    implementation "androidx.room:room-runtime:$roomVersion"
```

Comment:
Try to have some good structure, for example, `implementation`s should be grouped separate, `testImplementation` and `androidTestImplementation` separate (by separation I mean blank line). Keep `kapt` below the library it is used for.